### ISSUE TITLE
[Bug Report][2.6.0] A11y:  v-card title read out as clickable in NVDA when not expandable  

### ISSUE THREAD

**Issue by AlmeroSteyn** [vuetifyjs/vuetify#14447]
### Environment
**Vuetify Version:** 2.6.0
**Vue Version:** 2.6.11
**Browsers:** Chrome 96.0.4664.45
**OS:** Windows 10

### Steps to reproduce
Open NVDA screen reader.
Browse to any v-card with a v-card-title.
Use NVDA in browse mode with arrow navigation to navigate into the card and read out the title.

### Expected Behavior
Title is announced correctly as a non interactive text element.

### Actual Behavior
Title is announced as a clickable element.

### Reproduction Link
<a href="https://codepen.io/Kryptos_RSA/pen/ZEXEjPO" target="_blank">https://codepen.io/Kryptos_RSA/pen/ZEXEjPO</a>

### Other comments
This is caused by Vuetify adding an onClick handler to the wrapping div of the v-card-title. If this can be removed when not expandable the error will no longer appear.
This was tested with NVDA in both Chrome and Firefox.
This bug can confuse screen reader users into expecting the element to be interactive when it is not.
This was discovered in a project for the Dutch government where Vuetify is currently being used.

**Comment by KaelWD** [comment]
That seems like an NVDA bug, it's common to add a listener once then toggle it with a condition in the handler rather than continuously adding and removing it. I'd expect a div element to only be announced as clickable if it has something like `role="link"` or `role="button"`

**Comment by AlmeroSteyn** [comment]
@KaelWD Thank you for the quick response.

As far as I am aware this is intended behavior in NVDA. In order to assist SR users to also use pages that does not have ARIA applied correctly it has heuristics for determining when to announce and element as clickable and that is not solely based on the given role of the element. 

I have not tested it yet myself but others online mention JAWS exhibiting the same behavior. 

Here are some references related to this behavior:
https://github.com/microsoft/fluentui/issues/18795
https://github.com/nvaccess/nvda/issues/5830
https://github.com/facebook/react/issues/20895
https://github.com/carbon-design-system/carbon/issues/986
https://webaim.org/discussion/mail_thread?thread=7693

Thanks for the info. I was already aware that it is common practice to handle cases in the handlers themselves. However, in this case, the element is switching from interactive to fully display only. I am sure you would know the use cases better than me for hot switching between the two states but is it frequent enough that removing and re-attaching the handler only for this specific element would be problematic?

But the point is that SR intended behavior will announce all static Vuetify cards as interactive elements. It will not be blocking for users as they are used to a lot of this. But it is an annoyance and it would improve accessibility of the card if this can be changed.

Thanks again for taking the time to consider this.

**Comment by AlmeroSteyn** [comment]
Thanks for the quick action on this!!

I understand where you are coming from on this, however I don't think it is as simple to solve as stated in this comment. The assumption here is that there is something about the element or accessible name / text of the element that indicates it should be clickable. However in the wild this is far from certain.

It just adds more of a cognitive barrier then. Now it is a sea of clickables, but then it would be like one of those hidden object games instead. 

But the first prize would certainly be to get the web to a point where this is no longer required.
